app/ide-desktop/lib/dashboard/src/authentication/src/error.ts

```diff
@@ -54,7 +54,9 @@ export function getMessageOrToString<T>(error: MustNotBeKnown<T>) {
 /** Return a toastify option object that renders an error message. */
 // eslint-disable-next-line no-restricted-syntax
 export function render(f: (message: string) => string): toastify.UpdateOptions {
-    return { render: ({ data }) => f(getMessageOrToString(data)) }
+    return {
+        render: ({ data }: toastify.ToastContentProps<unknown>) => f(getMessageOrToString(data)),
```

Comment:
Again, types should match just fine here on their own. Make sure you have the dependencies installed correctly.

![image](https://github.com/enso-org/enso/assets/919491/61126d63-2400-4f61-8f3c-c3f38c22afdc)
![image](https://github.com/enso-org/enso/assets/919491/01849b86-b188-41c0-b87c-b27d6772a952)